src/components/components/ebay-tooltip-base/index.js

```diff
@@ -0,0 +1,52 @@
+const Expander = require('makeup-expander');
+const emitAndFire = require('../../../common/emit-and-fire');
+const template = require('./template.marko');
+
+function getInitialState(input) {
+    const state = Object.assign({}, input, {
+        location: input.location || 'bottom'
+    });
+
+    return state;
+}
+
+function init() {
+    const isHostPresent = this.el.querySelector(`.${this.state.type}__host`);
+
+    if (isHostPresent) {
+        this.expander = new Expander(this.el, {
+            hostSelector: `.${this.state.type}__host`,
+            contentSelector: `.${this.state.type}__overlay`,
+            focusManagement: this.state.type === 'tourtip' ? null : 'focusable',
```

Comment:
Let's not do any focus management when this thing opens. Let's keep focus on the host. A problem with setting focus on the close button of an Infotip is that if a screen reader navigates forward they will miss the content of the Infotip (due to the close button being at the end).

Reply:
👍  That's an easy change.